### PR TITLE
PAAS-2288: send logs to OVH as well

### PR DESCRIPTION
**Pull request by adoux**
JIRA issue: https://jira.jahia.org/browse/PAAS-2288

Short description:

### CODE REVIEW

File: mixins/common.yml [hunk 1]
```diff
@@ -1058,24 +1058,59 @@ actions:
...
+               -e "s#LOGS_SECRET_KEY_PLACEHOLDER#$__secret__secret_key#" $rclone_conf
 
-        sed -i "s/LOGS_ACCESS_KEY_PLACEHOLDER/$__secret__access_key/" $rclone_conf
-        sed -i "s/LOGS_SECRET_KEY_PLACEHOLDER/$__secret__secret_key/" $rclone_conf
-        sed -i "s/LOGS_REGION_PLACEHOLDER/$region/" $rclone_conf
+        if [ "$provider" = "ovh" ]; then
+          provider="Other"
+          endpoint="https://s3.$region.io.cloud.ovh.net/"
+          sed -i -e "s;LOGS_ENDPOINT_PLACEHOLDER;$endpoint;" \
+                 -e  "s#LOGS_AES256_KEY_PLACEHOLDER#$__secret__aes256_key#" \
```
Comment (amitc-jahia) [inline mixins/common.yml:1105]: Extra space between the option and the script

File: mixins/common.yml [hunk 2]
```diff
@@ -1058,24 +1058,59 @@ actions:
...
+                 -e  -e "/#PROVIDER_AWS/d" \
```
Comment (amitc-jahia) [inline mixins/common.yml:1106]: Remove the duplicate option "-e" from the same line

**Review by amitc-jahia — CHANGES_REQUESTED** [review]

File: mixins/common.yml [hunk 3]
```diff
@@ -1058,24 +1058,59 @@ actions:
...
+                 -e "s/ #PROVIDER_OVH//" $rclone_conf
+        else
+          provider=AWS
+          sed -i "/#PROVIDER_OVH/d" $rclone_conf \
+                 -e "s/ #PROVIDER_AWS//"
```
Comment (laurentfufu) [inline mixins/common.yml:1111]:
```suggestion
          sed -i -e "/#PROVIDER_OVH/d" \
                 -e "s/ #PROVIDER_AWS//" $rclone_conf
```
(to stick to the spirit, although personally I prefer the `-i <file>` in the first or last line)

**Review by laurentfufu — CHANGES_REQUESTED** [review]

File: mixins/common.yml [hunk 4]
```diff
@@ -1095,16 +1130,20 @@ actions:
         bucket_name=$(echo $organization | jq -r '.logs_s3_bucket')
         region_id=$(echo $organization | jq -r '.logs_s3_bucket_region_id')
         region_code=$(papi.py -X GET paas-region/$region_id | jq -r .code)
-        credentials=$(papi.py -X GET paas-organization/$organization_id/aws-secrets)
+        provider=$(papi.py -X GET paas-region/$region_id | jq -r .provider)
```
Comment (laurentfufu) [inline mixins/common.yml:1133]: it will be great to avoid several requests on the same endpoint

File: mixins/common.yml (hunk 1 above)
Comment (adoux) [reply mixins/common.yml]: Oops, good catch

File: mixins/common.yml (hunk 2 above)
Comment (adoux) [reply mixins/common.yml]: :man_facepalming: 
Done!

File: mixins/common.yml (hunk 3 above)
Comment (adoux) [reply mixins/common.yml]: Done!

File: mixins/common.yml (hunk 4 above)
Comment (adoux) [reply mixins/common.yml]: Good catch, fixed!

**Review by laurentfufu — APPROVED** [review]

**Pull request merged**